framework/harness/harness.go

```diff
@@ -14,41 +15,58 @@ import (
 	"github.com/launchdarkly/sdk-test-harness/v2/framework"
 )
 
-//go:embed certificate/cert.crt
+//go:embed certificate/leaf_public.pem
 var certificate []byte
 
-//go:embed certificate/cert.key
+//go:embed certificate/leaf_private.pem
 var privateKey []byte
 
+//go:embed certificate/ca_public.pem
+var caCertificate []byte
+
 type certPaths struct {
 	cert string
 	key  string
+	ca   string
```

Comment:
caPool or caFile may be a better name if this file can contain more than one CA.  I see ca and think this is something that can issue certs (i.e. has a private key for a ca)